clayui.com/src/components/Editor/index.js

```diff
@@ -38,39 +36,31 @@ const Editor = ({
 	preview = true,
 	scope = {},
 }) => {
-	let defaultSnippet = {
-		code: '',
-		disabled: false,
-		format: true,
-		imports,
-		name: 'React',
-	};
-
 	if (Array.isArray(code)) {
-		const codeToFormat = [...code];
-
-		codeToFormat.forEach((snippet) => {
-			if (snippet.format) {
-				snippet.code = formatCode(snippet.code);
-			}
+		code = code.map(({code, format = true, ...other}) => {
```

Comment:
defaulted `format` to true

Reply:
I also opted for using a `.map` so that we don't have to clone the array and mutate its items